tests/generation/test_stopping_criteria.py

```diff
@@ -127,7 +130,13 @@ def test_validate_stopping_criteria(self):
 
         self.assertEqual(len(stopping_criteria), 1)
 
-    def test_stop_string_criteria(self):
+    @parameterized.expand(
```

Comment:
(reuses the extensive tests for `StopStringCriteria ` on `StopStringTextMatchCriteria`, ensuring 1:1 compatibility)